### PR TITLE
Translation into FR

### PR DESCRIPTION
**Pull request by frju365**
Need review.

## Potential Problems

- String too long (first sentences)
- FS: translation of KiB to Kio ? (Others ?)

### CODE REVIEW

**Comment by andersevenrud** [comment]
All that's left here is to update the configuration file, and I can merge it it seems :)

File: src/locale/fr_FR.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  // Core
+  ERR_REQUEST_STANDALONE: 'Impossible d\'effectuer des requêtes en mode autonome.',
+  ERR_REQUEST_NOT_OK: 'Une erreur s\'est produite en exécutant une requête : {0}',
+  ERR_VFS_MOUNT_NOT_FOUND: 'Système de fichier \'{0}\' absent',
+  ERR_VFS_MOUNT_NOT_FOUND_FOR: 'Système de fichier pour \'{0}\' absent',
+  ERR_VFS_MOUNT_NOT_MOUNTED: 'Le système de fichier \'{0}\' n\'est pas encore monté',
+  ERR_VFS_MOUNT_ALREADY_MOUNTED: 'Le système de fichier \'{0}\' est déjà monté',
+  ERR_VFS_PATH_FORMAT_INVALID: 'Le dossier \'{0}\' ne correspond pas \'name:/path\'',
+  ERR_PACKAGE_NOT_FOUND: 'Les métadonnées du paquet \'{0}\' n\'ont pas été touvées',
+  ERR_PACKAGE_LOAD: 'Package Loading \'{0}\' failed: {1}',
```
Comment (andersevenrud) [inline src/locale/fr_FR.js]: Seems like you forgot one :)

Comment (andersevenrud) [reply src/locale/fr_FR.js]: Line 41 to be spesific.

Comment (frju365) [reply src/locale/fr_FR.js]: Done

**Comment by andersevenrud** [comment]
Great! Thanks :ok_hand:

**Pull request merged**